rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/Sw360ResourceServer.java

```diff
@@ -37,4 +40,15 @@ protected SpringApplicationBuilder configure(SpringApplicationBuilder builder) {
     public static void main(String[] args) {
         SpringApplication.run(Sw360ResourceServer.class, args);
     }
+
+    @Bean
+    public RepositoryRestConfigurer repositoryRestConfigurer() {
+        return new RepositoryRestConfigurerAdapter() {
+            @Override
+            public void configureRepositoryRestConfiguration(RepositoryRestConfiguration config) {
+                config.setLimitParamName("page_entries");
+                config.setDefaultPageSize(10);
```

Comment:
Should we make this value configurable?
e.g. add this to rest .yml or sw360.properties?